### PR TITLE
Replace pulseaudio with pipewire stack

### PR DESCRIPTION
**Pull request by Samsagax**
Pipewire is stable enough now to be used. Being using it for a long time on all my setups without issues.

### CODE REVIEW

**Comment by alkazar** [comment]
Pipewire does not work for me when plugging/unplugging headphones. Let's give it a shot on a clean system though.

File: manifest [hunk 1]
```diff
@@ -40,16 +40,16 @@ export PACKAGES="\
 	networkmanager \
 	pipewire \
 	lib32-pipewire \
+	pipewire-alsa \
+	pipewire-pulse \
+	pipewire-jack \
+	gst-plugin-pipewire \
```
Comment (alkazar) [inline manifest:46]: What is this for?

File: manifest [hunk 2]
```diff
@@ -40,16 +40,16 @@ export PACKAGES="\
 	networkmanager \
 	pipewire \
 	lib32-pipewire \
+	pipewire-alsa \
+	pipewire-pulse \
+	pipewire-jack \
```
Comment (alkazar) [inline manifest:45]: I don't think we need jack.

File: manifest [hunk 3]
```diff
@@ -40,16 +40,16 @@ export PACKAGES="\
 	networkmanager \
 	pipewire \
 	lib32-pipewire \
+	pipewire-alsa \
```
Comment (alkazar) [inline manifest:43]: Should we also install wireplumber?

File: manifest (hunk 1 above)
Comment (Samsagax) [reply manifest:46]: The gst-plugin is needed for streaming/recording gamescope session, gnome-shell or any other pipewire stream. It only adds 46Kb to the compressed image and 127Kb fully installed

File: manifest (hunk 2 above)
Comment (Samsagax) [reply manifest:45]: I've seen some packages depending on jack, but if you see it interfering, we can drop it.

File: manifest (hunk 3 above)
Comment (Samsagax) [reply manifest:43]: Wireplumber is installed as dependency of any of pipewire-{alsa,pulse,jack} packages.

**Pull request merged**